src/platform/intel/ace/lib/dma.c

```diff
@@ -92,6 +92,7 @@ SHARED_DATA struct dma dma[] = {
 		.devs		= DMA_DEV_SSP | DMA_DEV_DMIC |
 				  DMA_DEV_ALH,
 		.channels	= 8,
+		.period_count	= DW_DMA_BUFFER_PERIOD_COUNT,
```

Comment:
@softwarecki @kv2019i @marcinszkudlinski sounds like a topic for a future Zephyr API update. i.e. an API extension for this and similar uses cases.

Reply:
@lgirdwood The API was already extended actually for our needs, but this 'peroid-count' was left out as it was deemed too audio centric to have in generic DMA interface. Can be added later...